torch/linalg/__init__.py

```diff
@@ -2042,3 +2042,19 @@
     >>> torch.dist(Q.transpose(-2, -1) @ Q, torch.eye(4))
     tensor(6.2158e-07)
 """)
+
+trace = _add_docstr(_linalg.linalg_trace, r"""
+trace(input, offset=0) -> Tensor
+
+Returns the sum of the elements of the diagonal.
```

Comment:
What diagonal? Given that we have the parameter `offset`, this should probably read:

>Returns the sum of the elements of a diagonal.

Followed by an explanation of how the `offset` parameter chooses a diagonal.